packages/providers/src/sdk/CryptKeeperInjectedProvider.ts

```diff
@@ -386,4 +388,28 @@ export class CryptKeeperInjectedProvider {
       method: RPCAction.REVEAL_CONNECTED_IDENTITY_COMMITMENT_REQUEST,
     });
   }
+
+  /**
+   * Requests user to join a group with current connected identity.
+   *
+   * @returns {Promise<void>}
+   */
+  async joinGroup(payload: IJoinGroupMemberArgs): Promise<void> {
+    await this.post({
+      method: RPCAction.JOIN_GROUP_REQUEST,
+      payload,
+    });
+  }
+
+  /**
+   * Requests user to generate a group membership proof with current connected identity.
+   *
+   * @returns {Promise<void>}
+   */
+  async generateGroupMembershipProof(payload: IGenerateGroupMerkleProofArgs): Promise<void> {
```

Comment:
I think its better to call it `generateGroupMerkleProof` we need to have similar used terms as possible.

Reply:
This change would be fixed in another PR.